src/Service/Iam/src/Enum/statusType.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace AsyncAws\Iam\Enum;
+
+final class statusType
```

Comment:
This is strange. The AWS specification names this `statusType` and not `StatusType`. 

We can: 
1. Force first character in a class to be capitalised. 
2. Ignore it
3. Add a patch to the manifest.

Reply:
Will fix this globally in another PR

Reply:
FYI @jderusse

Reply:
This might be a fix: #815

Reply:
#815 merge. Could you please regenerate the method to apply the change @amenophis?